drivers/sensor/lis2dh/lis2dh_trigger.c

```diff
@@ -395,47 +407,47 @@ int lis2dh_init_interrupt(const struct device *dev)
 #endif
 
 	/* data ready int1 gpio configuration */
-	status = gpio_pin_configure(lis2dh->gpio_int1, LIS2DH_INT1_GPIOS_PIN,
-				    GPIO_INPUT | LIS2DH_INT1_GPIOS_FLAGS);
+	status = gpio_pin_configure(lis2dh->gpio_int1, cfg->irq1_pin,
+				    GPIO_INPUT | cfg->irq1_flags);
 	if (status < 0) {
-		LOG_ERR("Could not configure gpio %d",
-			     LIS2DH_INT1_GPIOS_PIN);
+		LOG_ERR("Could not configure gpio %d", cfg->irq1_pin);
 		return status;
 	}
 
 	gpio_init_callback(&lis2dh->gpio_int1_cb,
 			   lis2dh_gpio_int1_callback,
-			   BIT(LIS2DH_INT1_GPIOS_PIN));
+			   BIT(cfg->irq1_pin));
 
 	status = gpio_add_callback(lis2dh->gpio_int1, &lis2dh->gpio_int1_cb);
 	if (status < 0) {
 		LOG_ERR("Could not add gpio int1 callback");
 		return status;
 	}
 
-	LOG_INF("int1 on pin=%d", LIS2DH_INT1_GPIOS_PIN);
+	LOG_INF("int1 on %s%d", cfg->irq1_dev_name, cfg->irq1_pin);
```

Comment:
Mmmmh. Looks like GPIOA0, GPIOC0, GPIOC1. Something like that. But that because I'm using the shield on top of a nucleo board. Maybe I can do something like "GPIOA(0)" or "GPIOA (0)". What do you think about it?

Reply:
I personally like `%s.%02u` for peripheral+pin formatting.

Reply:
I completely forgot about this point. I will follow the suggestion of @pabigot, which seems a nice and clear way to represent this kind of resources.